README.md

```diff
@@ -7,7 +7,15 @@ In order to setup concourse initially on a cluster, you will need to have `terra
 
 1. Edit `resources/secrets.tf` and add a configuration block for the new cluster, if one does not already exist.
 
-2. `./bin/setup-on-cluster <cluster-name>` where `cluster-name` is the name of the cluster you want to install Concourse CI on, as it appears in the terraform workspaces [here](https://github.com/ministryofjustice/kubernetes-investigations/tree/master/terraform/cloud-platform).
+## Prerequisite before running step 2.
+a) Select the name of the cluster you want to install Concourse CI on, as it appears in the terraform workspaces [here](https://github.com/ministryofjustice/kubernetes-investigations/tree/master/terraform/cloud-platform).
+
+kubectl config use-context <cluster-name>
```

Comment:
Typically, this would be `<cluster-name>.k8s.integration.dsd.io`

Reply:
updated use-context to <cluster-name>.k8s.integration.dsd.io